### DISCUSSION TITLE
How can we use import statement in our wdio.config file

### DISCUSSION THREAD

**Discussion by divyagupta31** [webdriverio/codemod#81] [General]
Problem statement: Trying to use hooks for logging into the application. My login code is in one of the ts file. So
i want to import that ts file in my config file to use login methods.

what i am doing:
![image](https://user-images.githubusercontent.com/40352515/214266639-076ccab2-783c-4755-bdd6-40459471171f.png)
![image](https://user-images.githubusercontent.com/40352515/214266935-36433e0c-4c59-4a84-ae4a-f115c8208d73.png)

As of now i am facing this issue:
![image](https://user-images.githubusercontent.com/40352515/214267054-e8bf857a-fcb5-4239-bffa-e6288810ad35.png)

help here!!!!

Also if we can't use import ,what can be done here?

**Comment by christian-bromann** [comment]
> Also if we can't use import ,what can be done here?

I am not sure what you are doing here as your code examples don't really show anything but you might want to take a look into the [docs](https://webdriver.io/docs/api/globals).
